locktree/lock_request.cc

```diff
@@ -339,10 +377,6 @@ void lock_request::retry_all_lock_requests(locktree *lt) {
             i++;
         }
     }
-
-    // future threads should only retry lock requests if some still exist
-    info->should_retry_lock_requests = info->pending_lock_requests.size() > 0;
```

Comment:
@vlad-lesin  this line be removed after merge, but still exists in the latest master, as we can see here:
https://github.com/percona/PerconaFT/blob/43f81569cfd002763e080689d720e52265b72206/locktree/lock_request.cc#L382-L384